docs/src/tutorials/global_identifiability.md

```diff
@@ -0,0 +1,58 @@
+# Global Identifiability of Differential Models
+
+In this tutorial, let us cover an example problem of querying the ODE for globally identifiable parameters.
+
+## Input System
+
+Let us consider the following four-dimensional model with two outputs:
+
+$\begin{cases}x'(t) = lm - d * x(t) - beta * x(t) * v(t),\\
+    y'(t) = beta * x(t) * v(t) - a * y(t),\\
+    v'(t) = k * y(t) - u * v(t),\\
+    w'(t) = c * x(t) * y(t) * w(t) - c * q * y(t) * w(t) - b * w(t),\\
+    z'(t) = c * q * y(t) * w(t) - h * z(t),\\
+    y_1(t) = w(t),\\
+    y_2(t) = z(t)\end{cases}$
+
+This model comes from [[1]](#hiv), describing HIV dynamics. Let us run a global identifiability check on this model to get the result with probability of correctness being `p=0.999`.
+
+```@repl
+using StructuralIdentifiability
+
+ode = @ODEmodel(
+    x'(t) = lm - d * x(t) - beta * x(t) * v(t),
+    y'(t) = beta * x(t) * v(t) - a * y(t),
+    v'(t) = k * y(t) - u * v(t),
+    w'(t) = c * x(t) * y(t) * w(t) - c * q * y(t) * w(t) - b * w(t),
+    z'(t) = c * q * y(t) * w(t) - h * z(t),
+    y1(t) = w(t),
+    y2(t) = z(t)
+)
+@time global_id = assess_global_identifiability(ode, 0.999)
+```
+
+Now let us compare the same system but with probability being `p=0.99`. We will see a reduction in runtime:
+
+```@repl
+using StructuralIdentifiability
+
+ode = @ODEmodel(
+    x'(t) = lm - d * x(t) - beta * x(t) * v(t),
+    y'(t) = beta * x(t) * v(t) - a * y(t),
+    v'(t) = k * y(t) - u * v(t),
+    w'(t) = c * x(t) * y(t) * w(t) - c * q * y(t) * w(t) - b * w(t),
+    z'(t) = c * q * y(t) * w(t) - h * z(t),
+    y1(t) = w(t),
+    y2(t) = z(t)
+)
+@time global_id = assess_global_identifiability(ode, 0.99)
```

Comment:
As I wrote on Slack, this should be just `assess_identifiability` because `assess_global_identifiability` assumes that the input functions are locally identifiable and thus is not really intended to be used by a user